### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by vaxul** [projectdelphai/panorama-tab-groups#50]
We should translate all texts:
- title text for icons
- placeholder
- options

It should be a solution which would allow simple contributions by adding a new local. Similar to https://github.com/Quicksaver/Tab-Groups/tree/master/chrome/locale

**Comment by projectdelphai** [comment]
I have something for this in the works already. Once it's finished, I'll push it so that others can add translations.

**Comment by vaxul** [comment]
Would you share some insights, please? 🙂

**Comment by projectdelphai** [comment]
Localized strings in the view page should now be implemented. I added code to pull strings based on the language set by the browser if available or to default to English if not available. Since the options page is all hardcoded HTML, I wasn't able to do it since [these instructions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization) only account for strings provided via javascript. I found something like [this](https://github.com/erosman/HTML-Internationalization) which should allow us to replace html, but that can take more work.

For now, the stuff that needs to be done:

 1. Add instructions to the README on how other people can add translations
 2. Add ability to translate the options page
 3. Actually get translations for other pages
